playground/AzureFunctionsEndToEnd/AzureFunctionsEndToEnd.ApiService/Program.cs

```diff
@@ -14,8 +14,8 @@
 builder.AddServiceDefaults();
 builder.AddAzureQueueClient("queue");
 builder.AddAzureBlobClient("blob");
-#if !SKIP_EVENTHUBS_EMULATION
 builder.AddAzureEventHubProducerClient("eventhubs", static settings => settings.EventHubName = "myhub");
+#if !SKIP_AZURE_RESOURCE
```

Comment:
This name is too general. There are a bunch of azure resources we aren't skipping.

Maybe `SKIP_PROVISIONED_AZURE_RESOURCES`?

Reply:
Sure -- I can rename this.